frontend/src/components/IntroductionBox.tsx

```diff
@@ -3,9 +3,10 @@ import "./IntroductionBox.css"
 export default function IntroductionBox () {
    return(
        <p id="introductionBox">
```

Comment:
```suggestion
       <p className="introductionBox">
```
Man sollte className bevorzugt nutzen und nur wenn es anderes nicht möglich ist auf id zurück greifen.
Wenn du dass änderst, dann müsst du im css `.introductionBox` schreiben